config/schema/field_encrypt.schema.yml

```diff
@@ -14,6 +14,9 @@ field.storage.*.*.third_party.field_encrypt:
     encryption_profile:
       type: string
       label: 'Encryption profile'
+    cache_exclude:
+      type: boolean
+      label: 'Exclude from cache'
```

Comment:
I think `uncacheable` would be a better name. It'd also fit in better with the terminology in Drupal core.